src/devices/Display/samples/Large4Digit7SegmentDisplay.sample.cs

```diff
@@ -0,0 +1,201 @@
+                new KeyValuePair<string, Alignment>(":P", Alignment.Left),
+                new KeyValuePair<string, Alignment>(":PPP", Alignment.Left),
+                new KeyValuePair<string, Alignment>(":PPPP", Alignment.Left),
+                new KeyValuePair<string, Alignment>(":", Alignment.Left),
+            })
+            {
+                display.Write(s.Key, s.Value);
+                // Send buffer to the device
+                display.Flush();
+                // Wait .5 second
+                Thread.Sleep(500);
+            }
+
+            // Turn off all dots
+            display.Dots = Dot.Off;
+
+            var hexDigits = new byte[4];
+            new Random().NextBytes(hexDigits);
+
+            // Display random hex number
+            display.Write(FontHelper.GetHexDigits(hexDigits));
+
+            // Send buffer to the device
+            display.Flush();
+
+            // Wait 3 seconds
+            Thread.Sleep(3000);
+
+            // Display border
+            //  ---
+            // |   |
+            //  ---
+            display.Write(new[]
+            {
+                Segment.Top | Segment.TopLeft | Segment.BottomLeft | Segment.Bottom,
+                Segment.Top | Segment.Bottom,
+                Segment.Top | Segment.Bottom,
+                Segment.Top | Segment.TopRight | Segment.BottomRight | Segment.Bottom
+            });
+
+            // Send buffer to the device
+            display.Flush();
+
+            // Wait 3 seconds
+            Thread.Sleep(3000);
+
+            // Iterate supported characters
+            for (int i = 0, l = SupportedCharacters.Length - 3; i < l; i++)
+            {
+                display.Write(SupportedCharacters.Substring(i, 4));
+                // Send buffer to the device
+                display.Flush();
+                // Wait 0.5 seconds
+                Thread.Sleep(500);
+            }
+
+            // Iterate dots
+            foreach (var dot in new[] { Dot.CenterColon, Dot.DecimalPoint, Dot.LeftLower, Dot.LeftUpper })
+            {
+                display.Dots = dot;
+                // Send buffer to the device
+                display.Flush();
+                // Wait 0.5 seconds
+                Thread.Sleep(500);
+            }
+
+            // Set all supported dots
+            display.Dots = Dot.CenterColon | Dot.DecimalPoint | Dot.LeftLower | Dot.LeftUpper;
+
+            display.Flush();
+
+            // Wait 3 seconds
+            Thread.Sleep(3000);
+
+            // Write OFF to right side of display
+            display.Write("Off", Alignment.Right);
+            // Send buffer to the device
+            display.Flush();
+
+            // Wait 2 seconds
+            Thread.Sleep(2000);
+
+            // Turn off display
+            display.DisplayOn = false;
+
+            // Dispose display object (the device itself will not be turned off until powered down)
+            display.Dispose();
```

Comment:
use `using` statement instead (you can use the new syntax so that you don't need to indent)